app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -62,15 +63,15 @@ class SupplementalApplicationPage extends React.Component {
     this.setState({ persistedApplication: synchedApplication })
   }
 
-  handleSavePreference =  async (preferenceIndex, preferencesPanelApplication) => {
+  handleSavePreference =  async (preferenceIndex, application) => {
     const { persistedApplication } = this.state
 
-    // We clone the latest saved copy, so we can use the latest saved fields.
+    // We clone the lates saved copy, so we can use the latest saved fields.
```

Comment:
I realize this is closed now, but this update to change from non-typo to typo in a comment is pretty funny @fddayan

Reply:
wow 🤦‍♂️  not sure how that happened. I'll change that in my lates PR ready to merge.